src/Widgets/GalleryCarousel.jsx

```diff
@@ -27,7 +27,7 @@ class GalleryCarousel extends React.Component {
 
         {this.props.images.map((image, index) => {
           return (
-            <div className={index === this.state.currentImg ? 'slide active' : 'slide'} key={index}>
+            <div className={index === this.state.currentImg ? 'slide-active' : 'slide'} key={index}>
```

Comment:
why did you decide to do this?  perhaps you should keep the slide as a classname for ALL slides and add another classname called 'slide-active'? this allows you to style all styles, whether active or not, and also isolate the slide-active ones.